src/Microsoft.Azure.WebJobs.Host/Scale/ScaleManager.cs

```diff
@@ -0,0 +1,299 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.Host.Config;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Host.Scale
+{
+    /// <summary>
+    /// Class to get scaling votes for all the triggers or for an individual trigger.
```

Comment:
Any reason you changed the previous text "Manages scale monitoring operations."?

Reply:
Reverted back.